snarkos/cli.rs

```diff
@@ -48,18 +48,12 @@ pub struct CLI {
     #[clap(long = "connect")]
     pub connect: Option<String>,
 
-    /// Specify the IP address and port for the RPC server.
-    #[clap(parse(try_from_str), default_value = "0.0.0.0:3033", long = "rpc")]
-    pub rpc: SocketAddr,
-    /// Specify the username for the RPC server.
-    #[clap(default_value = "root", long = "username")]
-    pub rpc_username: String,
-    /// Specify the password for the RPC server.
-    #[clap(default_value = "pass", long = "password")]
-    pub rpc_password: String,
-    /// If the flag is set, the node will not initialize the RPC server.
+    /// Specify the IP address and port for the HTTP server.
+    #[clap(parse(try_from_str), default_value = "127.0.0.1:3033", long = "http")]
```

Comment:
```suggestion
    #[clap(parse(try_from_str), default_value = "0.0.0.0:3033", long = "http")]
```

We tried `127.0.0.1` and it does not work

Reply:
It doesn't work locally? I just tried
```
curl -X GET http://localhost:3033/testnet3/latest/hash
```
and it worked fine for me. IMO it's the safer default (it can still be set to `0.0.0.0` on startup), but it's your call; I'll add a commit